pkg/auth/rbac.go

```diff
@@ -33,41 +33,33 @@ import (
 type RBACConfig struct {
 	// Indicates if Dex is enabled.
 	DexEnabled bool
-	// The RBAC policy. Key is for example "p,Developer,EnvironmentLock,Create,production,allow"
-	Policy map[string]*Permission
+	Policy     map[string]*Permission
 }
 
 // Inits the RBAC Config struct
 func initPolicyConfig() policyConfig {
 	return policyConfig{
-		allowedApps:    []string{"EnvironmentLock", "EnvironmentApplicationLock", "Deploy", "Undeploy", "EnvironmentFromApplication"},
-		allowedActions: []string{"Create", "Delete"},
+		// List of allowed actions on the RBAC policy.
+		allowedActions: []string{
+			"CreateLock",
```

Comment:
These should be constants, e.g.
const (
	PermissionReleaseTrain = "ReleaseTrain"
)